### PR TITLE
CSVS-86 Functionality for handling creation of user and keeping tokens in CloudStack user tags was removed

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala [hunk 1]
```diff
@@ -50,11 +50,11 @@ object Tag {
...
 
   object Type {
-    case object User      extends Type
+    case object Account   extends Type
     case object UserVM    extends Type
 
     def toString(x: Type): String = x match {
-      case  Type.User       => "User"
+      case  Type.Account    => "Account"
       case  Type.UserVM     => "UserVM"
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala:58]: why does VM type have the "user" prefix? VM always belongs to a user

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/TestData.scala [hunk 2]
```diff
@@ -29,13 +29,12 @@ import com.bwsw.cloudstack.vault.server.common.WeightedQueue
...
 trait TestData {
-  val userId: UUID = UUID.randomUUID()
   val accountId: UUID = UUID.randomUUID()
   val vmId: UUID = UUID.randomUUID()
   val domainId: UUID = UUID.randomUUID()
 
 
-  val listUsersCommand = "listUsers"
+  val listAccountsCommand = "listAccounts"
   val listVms = "listVirtualMachines"
```
Comment (placccebo) [inline src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/TestData.scala]: is it a command too? as the above variable

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala:58]: I don't know, but it is written in https://cwiki.apache.org/confluence/display/CLOUDSTACK/Resource+Tags

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/TestData.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/TestData.scala]: it is not necessary, removed

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala (hunk 1 above)
Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/entities/TagResponse.scala:58]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**